<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <group ns="tj2" >
        <node pkg="tj2_pursuit" type="move_base_pursuit_sender.py" respawn="false" name="move_base_pursuit_sender" output="screen">
            <!-- <remap from="detections" to="cargo/detections"/> -->
            <remap from="detections" to="powercell/detections"/>
            <remap from="cmd_vel" to="cmd_vel_follow"/>
            <param name="detection_timeout" value="3.0"/>
            <param name="distance_offset" value="-0.5"/>
            <param name="distance_threshold" value="0.05"/>
            <param name="send_rate" value="5.0"/>
        </node>
    </group>
</launch>
